src/XMakeTasks/UnitTests/PortableTasks_Tests.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Build.Shared;
+using Microsoft.Build.SharedUtilities;
+using Xunit;
+
+namespace Microsoft.Build.UnitTests
+{
+    public sealed class PortableTasks_Tests
+    {
+        private static readonly string ProjectFilePath = Path.Combine(FileUtilities.CurrentExecutableDirectory, "portableTaskTest.proj");
```

Comment:
Maybe assert that this exists in `RunMSBuildOnProjectWithPortableTaskAndAssertOutput`? I don't think I'd have drawn the connection (I didn't even remember that it existed).